controller/argument_value_resolver.rst

```diff
@@ -81,8 +81,8 @@ with the ``User`` class::
         }
     }
 
-Beware that this feature is already provided by the `@ParamConverter`_
-annotation from the SensioFrameworkExtraBundle. If you have that bundle
+Beware that this feature is already provided by the `#[ParamConverter]`_
```

Comment:
Should we refer to the `MapEntity` (see symfony/symfony#43854) here instead?

Reply:
Actually, I would be in favor of changing the whole Custom Value Resolver example because there is already a native UserValueResolver. But I don't know what could be used instead!

Reply:
Let's keep it as is then for now and rethink the whole example in a different PR.